src/test/a11y/a11y.ts

```diff
@@ -0,0 +1,56 @@
+import {fail} from "assert";
+import * as promisify from "es6-promisify";
+import * as pa11y from "pa11y";
+import * as supertest from "supertest";
+import {app} from "../../main/app";
+
+const agent = supertest.agent(app);
+const pa11yTest = pa11y();
+const test = promisify(pa11yTest.run, pa11yTest);
+
+describe("Accessibility", () => {
+
+  // testing accessibility of the home page
+  testAccessibility("/");
+
+  // TODO: include each path of your application in accessibility checks
+});
+
+function testAccessibility(url: string): void {
```

Comment:
perhaps this could take a list of urls?

Reply:
I didn't want to go too far with actual implementation. I think the template should say `Look, this is how you can test accessibility for a page. Make sure you test all your pages.`. This function can be called in loops, modified or replaced by a different solution - it's up to the project team.

Reply:
fair enough